LabeledIconButton/LabeledIconButton.js

```diff
@@ -163,7 +166,10 @@ const LabeledIconButtonBase = kind({
  */
 const LabeledIconButtonDecorator = compose(
 	UiLabeledIconDecorator,
-	ButtonDecorator
+	Pure,
+	UiButtonDecorator,
+	Spottable,
+	Skinnable
```

Comment:
`ButtonDecorator` is replaced with 
```
	Pure,
	UiButtonDecorator,
	Spottable,
	Skinnable
```

By doing it, I'm worried that it is ok to remove `IconButtonDecorator`, `TooltipDecorator({tooltipDestinationProp: 'decoration'})`.

CC: @jeonghee27

Reply:
If LabeledIconButton don't need the iconOnly prop, IconButtonDecorator is not needed.
But if the tooltip support has not changed, let's keep TooltipDecorator.